Explorer/src/lib/db.ts

```diff
@@ -58,3 +58,12 @@ export async function getPodsList(): Promise<PodList> {
   const pods: PodList = res.rows;
   return pods;
 }
+
+export async function getPodDetails(id: string): Promise<Pod | undefined> {
+  return (
+    await pool.query(
+      "SELECT p.id, p.timestamp, p.name, p.pod_resource_version, p.pod_id, p.node_name, p.namespace, p.status_phase, p.host_ip, p.pod_ip, p.pod_ips, p.start_time, p.qos_class, c.container_id, c.image, c.status as container_status, c.ports, c.image_id FROM pods p JOIN containers c ON p.pod_id = c.pod_id WHERE p.id = $1 ORDER BY p.timestamp DESC LIMIT 1",
```

Comment:
`p.id` is the autogenerated id for the pod and should only be used with referenced id columns. `p.pod_id` is the actual pod uid from kubernetes which can be used to uniquely identify the resource independent of what state it currently is in.

Reply:
`WHERE p.id = $1` should be changed to `WHERE p.pod_id = $1`